aksk/pkg/service/service.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2021 The SODA Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"context"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/opensds/multi-cloud/aksk/pkg/iam"
+	"github.com/opensds/multi-cloud/aksk/pkg/model"
+	pb "github.com/opensds/multi-cloud/aksk/proto"
+)
+
+type akskService struct {
+}
+
+func NewAkSkService() pb.AkSkHandler {
+	return &akskService{}
+}
+
+func (b akskService) CreateAkSk(ctx context.Context, in *pb.AkSkCreateRequest, out *pb.AkSkCreateResponse) error {
```

Comment:
Returned object from ```NewAkSkService``` is pointer but defined receiver here is non pointer ?

Reply:
done